step-templates/redis-install.json

```diff
@@ -0,0 +1,71 @@
+{
+  "Id": "3cf1dadf-169d-4fd2-b95b-f5f07343b911",
+  "Name": "Redis - Install service",
+  "Description": "Script will make sure that Redis server is installed as a Windows service\n\n- It will start Redis inside package directory\n- To provide custom config you can prepare your own package (e.g. my-redis-64) or improve this template",
```

Comment:
This `Throw "$outputPath did not found in previous step. Make sure you point to 'Deploy package' type of step"` needs a bit of rewording, perhaps `Unable to find output path for step $step. Make sure you've selected the correct step for your package.`

Reply:
It was too late at night :P Thanks for pointing this one out.